### PR TITLE
VACMS-10661: Create report of outdated content currently in the CMS

### LINKED ISSUE #10661: Create report of outdated content currently in the CMS
## Background

To determine baseline metrics and phased rollout strategy for CMS notifications around VA directive 6102, we should run a report around what content in the CMS is currently outdated (more than 1 year old). The findings from this report could also help determine what sections to roll out notifications to first as a POC. Some ideas for the report include:

- filter by section
- filter by node
- filter by content type
- filter by product

## Acceptance Criteria

- [x] a CSV report of current outdated content more than one-year-old
- [x] a report view by node
- [x] a report view by section
- [x] report by product
- [x] report excludes events, stories, news releases
- [x] report is ONLY published content

### LINKED ISSUE #10840: Add past date filters to outdated content report
## Description

The MVP version of this report was completed as part of #10661. To iterate on this report for optimal functionality we should add date filters for:

- 30 days expired
- 60 days expired
- 90 days expired

We should also confirm the column for the dates to sort. 

## Acceptance Criteria
- [x] add 30/60/90 days past due filters
- [x] confirm date sorting column works accurately
- [x] confirm CSV date filters work

### CMS Team
Please check the team(s) that will do this work.

- [ ] `Program`
- [ ] `Platform CMS Team`
- [ ] `Sitewide Crew`
- [x] `⭐️ Sitewide CMS`
- [ ] `⭐️ Public Websites`
- [ ] `⭐️ Facilities`
- [ ] `⭐️ User support`

### PR DESCRIPTION
**Pull request by edmund-dunn**
## Description

Closes #10661.
Closes #10840.

## Testing done
Tested locally and on Tugboat. 

## Screenshots
CleanShot 2022-10-03 at 08 39 44

## QA steps
Navigate to main Content screen. Click on Outdate Content tab. You should see an OUtdated filter with "1 Year outdated" as the default. It should also have Over 30, Ovver 60 and over 90 days past due filters. The download CSV buttons hould also generate a CSV of all content that is past 1 year. 

### Select Team for PR review

- [ ] `Platform CMS Team`
- [ ] `Sitewide program`
- [x] `⭐️ Sitewide CMS`
- [ ] `⭐️ Public websites`
- [ ] `⭐️ Facilities`
- [ ] `⭐️ User support`

### CODE REVIEW

File: config/sync/core.extension.yml [hunk 1]
```diff
@@ -102,6 +103,10 @@ module:
   image_widget_crop: 0
   inline_entity_form: 0
   inline_form_errors: 0
+  jquery_ui: 0
+  jquery_ui_datepicker: 0
+  jquery_ui_slider: 0
+  jquery_ui_touch_punch: 0
```
Comment (ndouglas) [inline config/sync/core.extension.yml:109]: Did this all come in as a consequence or is it unrelated?

File: config/sync/views.view.content.yml [hunk 2]
```diff
@@ -14,6 +14,8 @@ dependencies:
     - node.type.faq_multiple_q_a
     - node.type.media_list_images
     - node.type.media_list_videos
+    - node.type.news_story
+    - node.type.press_release
```
Comment (ndouglas) [inline config/sync/views.view.content.yml:18]: holy diff, batman

**Review by ndouglas — COMMENTED** [review]
This is gonna take a while for me to review TBH.

File: config/sync/core.extension.yml (hunk 1 above)
Comment (edmund-dunn) [reply config/sync/core.extension.yml:109]: these are dependencies for better exposed filters

**Review by ndouglas — COMMENTED** [review]
Verified that no unexpected changes occurred to the admin content views.  All looks great!  Just need to update the spec tool 😞

**Comment by ndouglas** [comment]
```
      +++ Unexpected views displays (present in Drupal, absent from specification)
      | Content | Outdated Content CSV export | data_export_1 | Data export |
      | Content | Outdated content            | page_3        | Page        | (TravisCarden\BehatTableComparison\UnequalTablesException)
    │
```

**Review by ndouglas — APPROVED** [review]

**Pull request merged**